cmd/gendeepcopy/deep_copy.go

```diff
@@ -115,7 +115,15 @@ func main() {
 			generator.OverwritePackage(vals[0], vals[1])
 		}
 	}
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	var schemeVersion string
+	if version == "" {
+		// This occurs when we generate deep-copy for internal version.
+		schemeVersion = ""
+	} else {
+		// Trim "api/" from "api/v1", because "v1" is the groupVersion registered in the scheme.
```

Comment:
Yeah, same comment. I won't point it out everywhere. I don't want this logic duplicated in N places throughout our codebase.